back/engines/commercial/insights/spec/policies/insights/view_policy_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Insights::ViewPolicy, type: :policy do
+  subject { described_class.new(user, view) }
+
+  let!(:all_views) { create_list(:view, 2) }
+  let(:view) { all_views.first }
+  let(:scope) { described_class::Scope.new(user, Insights::View) }
+
+  context 'when user is admin' do
+    let(:user) { build(:admin) }
+
+    it { is_expected.to permit(:show) }
+    it { is_expected.to permit(:create) }
+    it { is_expected.to permit(:destroy) }
+    it { is_expected.to permit(:update) }
+    it { expect(scope.resolve.count).to eq(2) }
+  end
+
+  context "when user moderates the view's project" do
```

Comment:
Suggestion: "when user is moderator" - for consistency, though maybe there is a specific point being made here that I don't get.

Reply:
I note from the PR description that moderators can "... perform the same operations as admins, but only for their projects.", so maybe a better/alternative title is; "when user moderates their own project(s)"?

Reply:
Yep, today each view is associated to a project and a user can only see the view if (s)he moderates the associated project. What do you think of: *when user moderates the project associated to the view*, or is it even worse? 😄

Reply:
when user moderates the project associated _with_ the view 👍

Reply:
but they're all fine tbh